<!-- Launch File for running svo with ceres backend from bag-file -->
<launch>
  
  <arg name="type" default="vio"/>
  
  <arg name="calib_file" default="$(find svo_ros)/param/calib/realsense/realsense_triple.yaml"/>

  <!-- SVO node -->
  <node pkg="svo_ros" type="svo_visual_relocalization" name="svo_visual_relocalization" clear_params="true" output="screen" args="--v=0">

    <!-- Camera topic to subscribe to -->
    <param name="cam0_topic" value="/left30/cam0/image_raw" type="str" />
    <param name="cam1_topic" value="/left30/cam1/image_raw" type="str" />
    <!-- <param name="cam2_topic" value="/forward/cam1/image_raw" type="str" /> -->
    <param name="cam2_topic" value="/right30/cam1/image_raw" type="str" />
	
    <!-- Imu topic to subscribe to -->
    <param name="imu_topic" value="/imu0" type="str" />

    <!-- Config file -->
    <param name="config_file" value="$(arg config_file)" />

    <!-- Camera calibration file -->
    <param name="calib_file" value="$(arg calib_file)" />

    <!--Parameters-->
    <param name='runlc' value='false' />

  </node>

  <!-- RVIZ -->
  <!-- <node name="vis" pkg="rviz" type="rviz" args=" -d $(find svo_ros)/rviz_config_vio.rviz" /> -->

</launch>
